### PR TITLE
David/res spec fix

### PR DESCRIPTION
**Pull request by DavidSarlle**
@Edoardo-Pedicillo this branch fixes the bug found in the resonator spectroscopy PR #279. It would be nice if we merge this PR into #279 before merging into main.

I have tested and works as expected.

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by Edoardo-Pedicillo — COMMENTED** [review]
Thank @DavidSarlle, only one small comment

File: src/qibocal/protocols/characterization/resonator_spectroscopy.py [hunk 1]
```diff
@@ -133,8 +140,8 @@ def _acquisition(
     data = ResonatorSpectroscopyData(
         platform.resonator_type,
         params.power_level,
-        params.amplitude,
-        params.attenuation,
+        amplitudes, #params.amplitude,
+        attenuations, #params.attenuation,
```
[the remark this answers is not in the document]
Comment (DavidSarlle) [reply src/qibocal/protocols/characterization/resonator_spectroscopy.py]: Removed comments. Thanks @Edoardo-Pedicillo

**Review by Edoardo-Pedicillo — APPROVED** [review]

**Review by andrea-pasquale — APPROVED** [review]
Thanks @DavidSarlle, looks good to me.
Could you please update the docstrings?
We should specify to the user that if it specify an amplitude or attenuation value it should be the same for all qubits. Otherwise the default value will be used (which is qubit specific).
Also do you think that we should also provide the option to specify amplitude and attenuation at the same time? (Currently we are raising an error if this happens).

**Comment by DavidSarlle** [comment]
> Thanks @DavidSarlle, looks good to me. Could you please update the docstrings? We should specify to the user that if it specify an amplitude or attenuation value it should be the same for all qubits. Otherwise the default value will be used (which is qubit specific). Also do you think that we should also provide the option to specify amplitude and attenuation at the same time? (Currently we are raising an error if this happens).

@andrea-pasquale thanks for the comments. I have updated the docstrings in ResonatorSpectroscopyParameters class.

Regarding the possibility of giving the user the option of modifying the att and amp at the same time, I think it would be interesting in some cases. So the user can play finding the combination of both parameters to have the best RO. Alos, the code now, is applaying the default values if the params are not defined, so I do not see any problem if we are more flexible. What do you think?

**Comment by andrea-pasquale** [comment]
To me it is fine. I was just wondering because this option is currently forbidden by the code.
Could you please merge `split_data_fit` here to see if tests are passing?

**Comment by DavidSarlle** [comment]
> Tests are failing because `dummy` does not support attenuation. https://github.com/qiboteam/qibocal/actions/runs/4990897202/jobs/8936673296?pr=331 Now even if I try to run with the RFSoC I get the same error. Which makes sense, probably we need to modify something in the [`DesignPlatform`](https://github.com/qiboteam/qibolab/blob/a13fd1f5450156bbf6b6b93499e418692349b82e/src/qibolab/platforms/platform.py#L6) or make this routine more abstract.

@andrea-pasquale I do not understand why this branch is not passing he test and the split_data_fit did it. What shall we do in order to pass them? The modified routine was already present in the branch and I do not understand how the fixes affect to the test

**Pull request merged**